src/dmd/dsymbolsem.d

```diff
@@ -3897,6 +3897,12 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
     override void visit(NewDeclaration nd)
     {
         //printf("NewDeclaration::semantic()\n");
+
+        // @@@DEPRECATED_2019-05@@@
```

Comment:
We recently switched to using an explicit version, see also https://github.com/dlang/DIPs/pull/108 or https://github.com/dlang/phobos/pull/6286 for an example